x-pack/plugins/aiops/public/components/log_categorization/category_table/category_table.tsx

```diff
@@ -218,6 +220,29 @@ export const CategoryTable: FC<Props> = ({
     },
   ] as Array<EuiBasicTableColumn<Category>>;
 
+  if (displayExamples === false) {
+    // on the rare occasion that examples are not available, replace the examples column with tokens
+    columns.splice(2, 1, {
+      name: (
+        <EuiToolTip
+          position="top"
+          content={i18n.translate('xpack.aiops.logCategorization.column.tokens.tooltip', {
+            defaultMessage:
+              'If the selected field is an alias, example documents cannot be displayed. Showing pattern tokens instead.',
```

Comment:
Do we know for certain that the field is an alias? If so, this message could be:

```
Example documents cannot be displayed for alias field types. Showing pattern tokens instead.
```

Reply:
We don't know for certain, in my original version I was loading the mappings to determine this, but it seemed a bit unnecessary to do this seeing as this should rarely be hit.